### PR TITLE
Responsive account, update ui

### PR DESCRIPTION
**Pull request by ekiuse**

### CODE REVIEW

File: src/ui/gridContainer/gridContainer.module.css [hunk 1]
```diff
@@ -1,3 +1,4 @@
 .container {
```
Comment (nexnema) [inline src/ui/gridContainer/gridContainer.module.css:1]: change name to gridContainer

File: src/ui/header/header.module.css [hunk 2]
```diff
@@ -1,7 +1,9 @@
 .appBar {
```
Comment (nexnema) [inline src/ui/header/header.module.css:1]: change name

File: src/ui/header/header.tsx [hunk 3]
```diff
@@ -2,24 +2,11 @@ import React from "react";
 import { joinClassNames, joinStyles } from "library";
 import styles from "./header.module.css";
 
-interface HeaderProps extends React.HTMLAttributes<HTMLElement> {
-    direction?: "column" | "column-reverse" | "row" | "row-reverse";
-    gap?: number;
-}
-
-function Header(props: HeaderProps) {
+function Header(props: React.HTMLAttributes<HTMLElement>) {
```
Comment (nexnema) [inline src/ui/header/header.tsx:5]: do we have <HTMLHeaderElement>?

File: src/ui/svgIcon/svgIcon.module.css [hunk 4]
```diff
@@ -25,3 +25,47 @@
     fill: none;
     stroke: var(--onHeaderColor);
 }
+
```
Comment (nexnema) [inline src/ui/svgIcon/svgIcon.module.css:28]: add a task in project, to change SvgIcon entirely

**Review by nexnema — CHANGES_REQUESTED** [review]

File: src/ui/header/header.tsx (hunk 3 above)
Comment (ekiuse) [reply src/ui/header/header.tsx:5]: What should I change?

Comment (nexnema) [reply src/ui/header/header.tsx:5]: do we have HTMLHeaderElement?

**Review by nexnema — APPROVED** [review]

**Pull request merged**